### PR TITLE
Attempt to add logo to flat-square template

### PR DESCRIPTION
**Pull request by moigagoo**
I'm really not sure this would work, I just copied a code chunk from flat.

### CODE REVIEW

File: templates/flat-square-template.svg [hunk 1]
```diff
@@ -4,7 +4,12 @@
     <rect x="{{=it.widths[0]}}" width="{{=it.widths[1]}}" height="20" fill="{{=it.escapeXml(it.colorB||"#4c1")}}"/>
   </g>
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
-    <text x="{{=it.widths[0]/2+1}}" y="14">{{=it.escapeXml(it.text[0])}}</text>
+    {{?it.logo}}
+      <image x="5" y="3" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
```
Comment (espadrine) [inline templates/flat-square-template.svg]: Your use of `xlink` warrants the addition of a `xmlns:xlink="http://www.w3.org/1999/xlink"` in the SVG tag.

File: templates/flat-square-template.svg [hunk 2]
```diff
@@ -4,7 +4,12 @@
...
+    {{?}}
+    <text x="{{=(it.widths[0]+it.logoWidth+it.logoPadding)/2}}" y="15" fill="#010101" fill-opacity=".3">{{=it.escapeXml(it.text[0])}}</text>
+    <text x="{{=(it.widths[0]+it.logoWidth+it.logoPadding)/2}}" y="14">{{=it.escapeXml(it.text[0])}}</text>
+    <text x="{{=it.widths[0]+it.widths[1]/2-1}}" y="15" fill="#010101" fill-opacity=".3">{{=it.escapeXml(it.text[1])}}</text>
```
Comment (espadrine) [inline templates/flat-square-template.svg]: This is a shadow. Remove it; this style was designed not to have one.

File: templates/flat-square-template.svg [hunk 3]
```diff
@@ -4,7 +4,12 @@
     <rect x="{{=it.widths[0]}}" width="{{=it.widths[1]}}" height="20" fill="{{=it.escapeXml(it.colorB||"#4c1")}}"/>
   </g>
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
-    <text x="{{=it.widths[0]/2+1}}" y="14">{{=it.escapeXml(it.text[0])}}</text>
+    {{?it.logo}}
+      <image x="5" y="3" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
+    {{?}}
+    <text x="{{=(it.widths[0]+it.logoWidth+it.logoPadding)/2}}" y="15" fill="#010101" fill-opacity=".3">{{=it.escapeXml(it.text[0])}}</text>
```
Comment (espadrine) [inline templates/flat-square-template.svg]: This is a shadow. Remove it; this style was designed not to have one.

**Comment by espadrine** [comment]
Thanks for your contribution!

**Comment by moigagoo** [comment]
@espadrine Hi! I removed the shadow and added the xlink declaration.

**Comment by espadrine** [comment]
Thanks for your contribution! The change is live.

**Pull request merged**